.github/workflows/docker-publish.yml

```
@@ -0,0 +1,127 @@
+        with:
+          images: ${{ secrets.DOCKER_ORG }}/eloqkv
+          tags: |
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=raw,value={{date 'YYYY-MM-DD'}},enable=${{ github.event_name == 'workflow_dispatch' }}
+            type=raw,value=latest,enable=${{ github.ref_type == 'tag' }}
+
+      - name: Generate suffixed tags
+        id: suffixed_tags
+        shell: bash
+        run: |
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2086:info:2:21: Double quote to prevent globbing and word splitting

(shellcheck)